spec/services/secure_headers_allow_list_spec.rb

```diff
@@ -6,21 +6,6 @@ def csp_with_sp_redirect_uris(domain, sp_redirect_uris)
       SecureHeadersAllowList.csp_with_sp_redirect_uris(domain, sp_redirect_uris)
     end
 
-    it 'generates the proper CSP array from action_url domain and ServiceProvider#redirect_uris' do
```

Comment:
I meant to remove this test in a later commit. It is superfluous with the new tests below.